specification/compute/resource-manager/Microsoft.Compute/stable/2021-04-01/compute.json

```diff
@@ -11256,6 +11259,20 @@
       },
       "description": "Describes a scale-in policy for a virtual machine scale set."
     },
+    "SpotRestorePolicy": {
+      "type": "object",
```

Comment:
@ArcturusZhang whats the difference between defining type as object here and not defining type at all ?

Reply:
@Sandido @sandeepkv93

Reply:
One of the automated checks was mentioning to add the object type

Reply:
L10267 for automaticRepairsPolicys looks identical as SpotRestorePolicy except the type. Is this something new that is being enforced? @ArcturusZhang

Reply:
Defining no type at all is equivalent to defining it as `type: object`. The modeler will give you a warning about `no type defined` and automatically correct your swagger as `type: object`. Therefore it is very recommended to make it explicit - define `type: object` here.

Reply:
thank you @ArcturusZhang